Common/src/main/java/com/github/sef24sp4/common/interfaces/IGameSettings.java

```diff
@@ -10,6 +10,11 @@ public interface IGameSettings {
 
 	public int getDisplayHeight();
 
+	public void setDisplayWidth(int displayWidth);
+
+	public void setDisplayHeight(int displayHeight);
+
+
```

Comment:
These method should not be on `IGameSettings` the display width and height is inherently readonly.